dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -29,11 +29,11 @@ class CoursesControllerTest < ActionController::TestCase
 
   # Tests for index
 
-  test_user_gets_response_for :index, response: :success, user: :teacher, queries: 12
+  test_user_gets_response_for :index, response: :success, user: :teacher, queries: 4
```

Comment:
Since we now have the family name for course A-F, express, and pre-express in the fixtures that information gets pulled from the cache instead so all 3 of these tests query count reduced by 8.